comparators/README.md

```diff
@@ -0,0 +1,23 @@
+# Scope of current comparators
+
+* **added-website**  : Reports true when a `website` tag has been added, deleted or modified in a feature.
+
+* **disputed-border-deleted** : Reports true when a feature with `disputed` tag in it, is deleted.
+
+* **disputed-border-tag-changed** : Reports true when a feature with `disputed` tag has been modified (Property / geometry).
```

Comment:
`Reports when the geometry or tags of a feature with `disputed` tag has been modified.`